cmd/prometheus/main.go

```diff
@@ -229,6 +229,10 @@ func main() {
 		"Maximum duration compacted blocks may span. For use in testing. (Defaults to 10% of the retention period.)").
 		Hidden().PlaceHolder("<duration>").SetValue(&cfg.tsdb.MaxBlockDuration)
 
+	a.Flag("storage.tsdb.max-chunk-size",
+		"The maximum size for a single chunk. Example: 512MB").
```

Comment:
Let's mention if we mean the WAL chunk segment or Block chunk segment or both?

Reply:
We already have a config option for WAL segment size in TSDB, an it should not be both anyway since we might want to tune both differently :) but yes, some mentioning.